src/igvfd/searches/configs/measurement_set.py

```diff
@@ -85,43 +88,44 @@ def measurement_set():
             'audit.INTERNAL_ACTION.category': {
                 'title': 'Audit Category: Internal Action'
             },
+            'type': {
+                'title': 'Object Type',
+            },
         },
         'facet_groups': [
             {
-                'title': 'Sample',
-                'facet_fields': [
-                    'donors.taxa',
-                    'samples.classifications',
-                    'samples.sample_terms.term_name',
-                    'samples.targeted_sample_term.term_name',
-                    'samples.disease_terms.term_name',
-                ],
-            },
-            {
-                'title': 'Assay',
+                'title': 'Measurement Set Details',
                 'facet_fields': [
                     'assay_term.term_name',
                     'preferred_assay_title',
-                    'samples.modifications.modality',
-                    'file_set_type',
+                    'control_type',
+                    'functional_assay_mechanisms.term_name',
                     'auxiliary_sets.file_set_type',
                     'samples.construct_library_sets.file_set_type',
                     'targeted_genes.symbol',
                     'samples.construct_library_sets.small_scale_gene_list.symbol',
-                    'functional_assay_mechanisms.term_name'
+                    'files.sequencing_platform.term_name',
+                    'externally_hosted',
```

Comment:
`externally_hosted` not in the `facets` object.

Reply:
removed, i need further discussion with Cricket.